examples/todo/src/main.rs

```diff
@@ -45,28 +51,26 @@ fn new(todo_form: Form<Todo>, conn: DbConn) -> Flash<Redirect> {
     let todo = todo_form.into_inner();
     if todo.description.is_empty() {
         Flash::error(Redirect::to("/"), "Description cannot be empty.")
-    } else if Task::insert(todo, &conn) {
-        Flash::success(Redirect::to("/"), "Todo successfully added.")
+    } else if let Err(e) = Task::insert(todo, &conn) {
+        Flash::error(Redirect::to("/"), &format!("Database error: {}", e))
```

Comment:
What do these database error messages actually look like? In a real application, you likely wouldn't want to divulge database information to the public, and so I worry that this is encouraging bad practice, depending on what information is actually contained in the error. Instead, we probably just want to log the error for private access.